modules/lobbies/scripts/find.ts

```diff
@@ -27,6 +29,16 @@ export async function run(
 	ctx: ScriptContext,
 	req: Request,
 ): Promise<Response> {
+	await ctx.modules.captcha.guard({
+		key: "lobbies.find",
+		period: 15,
```

Comment:
period & requests should be configurable in config.ts and have defaults in the default config